cg/models/cg_config.py

```diff
@@ -421,6 +429,15 @@ def housekeeper_api(self) -> HousekeeperAPI:
             self.housekeeper_api_ = housekeeper_api
         return housekeeper_api
 
+    @property
+    def janus_api(self) -> JanusAPIClient:
+        janus_api = self.__dict__.get("janus_api_")
+        if janus_api is None:
+            LOG.debug("Instantiating janus api")
+            janus_api = JanusAPIClient(config=self.dict())
```

Comment:
Consider only passing the specific data the janus client needs instead of everything. It is better to be explicit when composing an object

Reply:
I think we have this on the cg-ep for the next meeting. Right now this follows convention how other apis are instantiated into the CGConfig.